authors/apps/articles/tests.py

```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
```

Comment:
Are you using this file in your implementation?

Reply:
I am not using that file. Let me delete it.l

Reply:
File not in use